lang/en/tool_excimer.php

```diff
@@ -68,6 +68,7 @@
 $string['field_referer'] = 'Referer';
 $string['field_cookies'] = 'Cookies Enabled';
 $string['field_buffering'] = 'Buffering Enabled';
+$string['field_delete'] = '';
```

Comment:
Added but not used? Please remove or clarify where this is needed. Also it's empty, I don't think this should be here at all in this case.

Reply:
It's added to the table as part of a loop. If it is not there, then some means of determining whether or not there is a heading to display needs to be implemented. I figured this is the simplest.

Reply:
Actually seeing this makes me think this should be renamed to 'Actions' see previous comment

An example in core to copy:

With a menu:
http://moodle.local/admin/tool/analytics/index.php

With icons:
http://moodle.local/admin/searchareas.php